packages/mobile/e2e/src/utils/twilio.js

```diff
@@ -0,0 +1,42 @@
+import { TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, VERIFICATION_PHONE_NUMBER } from '@env'
+import twilio from 'twilio'
+import { sleep } from './utils'
+
+const MAX_TRIES = 120
```

Comment:
This is getting called just as the start button is pressed. How do you feel about increasing it to 3 minutes instead of 2 to give the SMS codes more time to arrive?

Reply:
Increased timeout to 3 minutes in https://github.com/valora-inc/wallet/pull/534/commits/e2fd0d47293a2784f6d92b1a54a9b9fce7c2982d.